pulpcore/cli/core/context.py

```diff
@@ -112,13 +117,20 @@ def scope(self) -> Dict[str, Any]:
 class PulpGroupContext(PulpEntityContext):
     ENTITY = _("user group")
     ENTITIES = _("user groups")
-    HREF = "auth_group_href"
+    # Handled by a workaround
+    # HREF = "group_href"
```

Comment:
I'm not a cli person so feel free to ignore. Should this stay in the final PR?

Reply:
Yes, because someone may miss this line. Also its supposed to be easy to remove the workaround again eventually.